### ISSUE TITLE
Add a "suggested tags" tab that shows children of existing tags.

### ISSUE THREAD

**Issue by SwordfishBooks** [hydrusnetwork/hydrus#901]
For example, if an image is tagged "series:superman" this tab would show:

character:clark kent
character:jimmy olsen
character:lex luthor
character:superman
character:lois lane
etc. 

which would make it easier to add missing character tags to the image.

**Comment by HunHage** [comment]
This already exists, but I believe it is disabled by default.
![image](https://user-images.githubusercontent.com/60720515/121289340-60dca300-c8aa-11eb-8da8-d5dcbebf2e42.png)

It will then appear here on the manage tags page.
![image](https://user-images.githubusercontent.com/60720515/121289372-718d1900-c8aa-11eb-9b5b-b0861b260ccc.png)

**Comment by SwordfishBooks** [comment]
I already have the "related" tab enabled.  It displays "statistically related tags based on what [the] focused file already has". 

When I tested it with a file with the sole tag "series:superman". It failed to even suggest "character:superman". Conversely, it suggested both "series:pokemon" and "series:pokémon".

**Comment by TheElo** [comment]
> 
> 
> I already have the "related" tab enabled. It displays "statistically related tags based on what [the] focused file already has".
> 
> When I tested it with a file with the sole tag "series:superman". It failed to even suggest "character:superman". Conversely, it suggested both "series:pokemon" and "series:pokémon".

It currently only works (and yes it actually already works) with hardtags. So if you have bunch of files that are tagged "series:superman" AND (!!) "character:superman" than files with one of those missing get the other recommended. The more tags are shared the higher the chance for getting the correct output.

Virtual tags are currently fully ignored by the related tags system from my impression.

Instead of a additional tab it would be also just as appropriate as include virtual tags in the tag search of the tag manager.

**Comment by Zweibach** [comment]
Yeah, this really doesn't need to be its own tab. Just have recommendations take into account virtual tags like parents.
